### PR TITLE
chore: format tests to fix lint errors

### PR DESCRIPTION
**Pull request by arbrown**
No actual changes. Simply the result of running `black tests` in the root repo to fix some lint errors.

### CODE REVIEW

**Review by losalex — APPROVED** [review]

File: tests/java/test_appengine_standard.py [hunk 1]
```diff
@@ -17,6 +17,7 @@
 
 from ..common.common import Common
 
+
```
Comment (minherz) [inline tests/java/test_appengine_standard.py:20]: nit: not needed space IMHO

File: tests/java/test_cloudrun.py [hunk 2]
```diff
@@ -19,6 +19,7 @@
 
 from ..common.common import Common
 
+
```
Comment (minherz) [inline tests/java/test_cloudrun.py:22]: nit: not needed space IMHO

File: tests/java/test_compute.py [hunk 3]
```diff
@@ -18,6 +18,7 @@
 
 from ..common.common import Common
 
+
```
Comment (minherz) [inline tests/java/test_compute.py:21]: nit: not needed space IMHO

File: tests/nodejs/test_cloudrun.py [hunk 4]
```diff
@@ -22,6 +22,7 @@
 
 from ..common.common import Common
 
+
```
Comment (minherz) [inline tests/nodejs/test_cloudrun.py:25]: nit: not needed space IMHO

File: tests/nodejs/test_kubernetes.py [hunk 5]
```diff
@@ -20,6 +20,7 @@
 
 from ..common.common import Common
 
+
```
Comment (minherz) [inline tests/nodejs/test_kubernetes.py:23]: nit: not needed space IMHO

**Review by minherz — APPROVED** [review]
lgtm. there are few extra spaces that not needed IMHO. :massage: If they were added automatically

File: tests/java/test_appengine_standard.py (hunk 1 above)
Comment (arbrown) [reply tests/java/test_appengine_standard.py:20]: Yes, they were added by the same tool (black) that was causing the lint test to fail.  I just ran the formatter over the files and this is what it did. 🤷‍♂️

**Pull request merged**